### ISSUE TITLE
Not getting text messages when app is not on recent items

### ISSUE THREAD

**Issue by virtualstacy** [CakeKing64/P5IM-android-10-fix#4]
I'm on Android 10, ASUS ROG Phone II. I don't receive text messages when the app is not on the recent items. And I also noticed that I would get them very late compared to when using other sms apps or even the default one. All permissions are allowed, and it's also the default one on the device's settings. I tried all of the releases here so far, even the ignoring_the_problem one.

**Comment by CakeKing64** [comment]
After a bit of testing I wasn't able to figure out how to recrate your not receiving messages problem (even with my app forcibly stopped it'd still get messages, so it might just be an ROG Phone II thing and cus pretty much all phones are different i'd have to have it with me in order to really figure out the problemo)

same goes with the delay in messages, but I can't really reliably test that atm (you can thank australia's amazing reception for that)

If I ever figure it out I'll get back to you but don't rely on it

**Comment by virtualstacy** [comment]
Hey thanks for responding, I appreciate that. I do suspect as much, and I have really no idea what else I could try apart from what was suggested here by you and others.

**Comment by CakeKing64** [comment]
If you haven't tried it already you can give 
https://zentalk.asus.com/en/discussion/comment/32898/#Comment_32898
a shot, maybe it might work, no bloody clue but it's worth a shot

But other than that hope you can fix it

**Comment by virtualstacy** [comment]
Heyy so it took me a while to get back here, been busy with things. But, I managed to figure out how to make it work for my device.  Battery Optimization didn't do anything. But after messing around with the settings, I got it to work normally by enabling the app under Auto-start Manager. I know most people don't run into this, but just in case someone else out there has the same device as mine, sharin' the love and the fix/workaround!
